crates/language/src/lib.rs

```diff
@@ -46,7 +46,7 @@ pub use ast_grep_core::Language;
 /// this macro implements bare-bone methods for a language
 macro_rules! impl_lang {
   ($lang: ident, $func: ident) => {
-    #[derive(Clone, Copy)]
+    #[derive(Clone, Copy, Debug)]
```

Comment:
Is Debug required here? A lot of other config structs do not have Debug impl.

Reply:
IMO essentially everything should be `Debug`. Especially public types.

Reply:
I would suggest add it in another pull request to make all public interface debuggable. Making Lang itself `Debug` does not make life easier. 

Instead, a separate PR to make config object debuggable will be a more systematic approach.

Reply:
Do you want me to remove the Debug/Display impls then? Keeping it gives specific langs better parity with `SupportLang`, which was my goal.

I also disagree it doesn't make anyone's life easier. It's generally expected any struct impls `Debug` unless there's a good reason for it not to, and not impling it infects any struct that contains it.

Reply:
> Keeping it gives specific langs better parity with SupportLang, which was my goal.

Sounds reasonable. Let's keep Debug and Display